client/styles/globals.css

```diff
@@ -1,16 +1,134 @@
-html,
-body {
-  padding: 0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+:root {
+  --family-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  --family-sans: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, var(--family-emoji);
 }
+:root,
+[color-scheme="light"] {
+    --body-bg: #f2f4f8;
+    --body-color: #343a3f;
+    --heading-color: #121619;
+    --link-color: #0043ce;
+    --link-visited: #6929c4;
 
-a {
-  color: inherit;
-  text-decoration: none;
+    --btn-bg: #4d5358;
+    --btn-text: #ffffff;
+    --btn-border: #343a3f;
+    --btn-focus-bg: #21272a;
+
+    --btn-primary-bg: #198038;
+    --btn-primary-text: #ffffff;
+    --btn-primary-border: #044317;
+    --btn-primary-focus-bg: #022d0d;
+
+    --btn-danger-bg: #da1e28;
+    --btn-danger-text: #ffffff;
+    --btn-danger-border: #750e13;
+    --btn-danger-focus-bg: #520408;
+
+    --btn-logout-bg: #007d79;
+    --btn-logout-text: #ffffff;
+    --btn-logout-border: #004144;
+    --btn-logout-focus-bg: #022b30;
+
+    --btn-toggle-bg: #7f8c8d;
+    --btn-toggle-text: #ffffff;
+    --btn-toggle-border: #34495e;
+    --btn-toggle-focus-bg: #53626f;
+
+    --group-border: #121619;
+
+    --input-bg: #ffffff;
+    --input-border: #6a737d;
+    --input-focus-border: #2188ff;
+
+    --label-hover: #c8e1ff;
+
+    /* Paginator */
+
+    --page-list-bg: #ffffff;
+    --page-list-border: #c1c1c1;
+
+    --thumbnail-bg: #dde1e6;
+    --thumbnail-label: #121619;
+    --thumbnail-focus-bg: #82cfff;
+    --thumbnail-number-bg: #ffffff;
+    --thumbnail-number-text: #4d5358;
+    --thumbnail-preview-border: #000000;
+
+    /* Editor */
+
+    --object-status-active: #bef264;
```

Comment:
I think the status colors need to be more clearly defined -- the active/inactive need to be distinct from each other so you can tell them apart at a glance, but these new colors make them look too similar:

![image](https://github.com/FalveyLibraryTechnology/VuDL/assets/309069/f0341f2f-2963-4024-b4dd-c7d03432c7ac)

Is there a reason we can't keep the standard red/green/yellow?